staging/src/k8s.io/apiextensions-apiserver/test/integration/validation_test.go

```diff
@@ -889,7 +892,12 @@ spec:
 		unexpectedViolations                  []string
 	}
 	tests := []Test{
-		{"empty", "", "", "", nil, nil, nil, nil},
+		{
+			desc: "empty",
+			expectedViolations: []string{
+				"spec.preserveUnknownFields: Invalid value: true: must not be true",
```

Comment:
can you make preserveUnknownFields a test input and add some test with and without?

Reply:
I added preserveUnknownFields as test input